### ISSUE TITLE
can't delete codemark in vscode

### ISSUE THREAD

**Issue by mikestaub** [TeamCodeStream/CodeStream#60]
**Describe the bug**
I can't delete a code mark. I can archive it.

**System Info (please complete the following information):**
 - 1.0.1
 - OSX 10.14.4
 - VS Code 1.34.0

**Log Files**
There is nothing in the logs.

**Comment by planteater** [comment]
Hi Mike - I wasn't able to reproduce the issue. Just to confirm, you're deleting the codemark from within CodeStream, and not from Slack, right? Assuming that's the case, can you go ahead and set the log info anyway? Might reveal something to our devs.

- Go into your VS Code user settings and search for CodeStream.
- Scroll down to the Trace Level setting and make sure it's set to "Debug".
- Reload VS Code and then try to reproduce the issue.
- Open the Output view (“View: Toggle Output” from the command palette) and select "CodeStream" from the dropdown menu at the top-right. Copy all of the output you see and save it in a text file.
- Change the dropdown select to “CodeStream (Agent)” and grab that output as well.

Thanks!

**Comment by mikestaub** [comment]
Actually, there are some errors in the logs, see here:

```
[2019-05-17 19:08:04:714] AgentConnection.sendRequest(codestream/post/delete) — {"postId":"CJT219Z7X|1558117830.001800","streamId":"CJT219Z7X"}
Error: Request codestream/post/delete failed with message: Unable to find message with id=1558117830.001800
[2019-05-17 19:08:04:714] /Users/mikestaub/.vscode/extensions/codestream.codestream-1.0.1/dist/extension.js:1:12236
Error: Request codestream/post/delete failed with message: Unable to find message with id=1558117830.001800
[2019-05-17 19:08:04:715] WebviewPanel: Completed posting response(wv:36:ksd2jDf2TBOhAzid7dfxzA) to the webview
[2019-05-17 19:08:04:716] Webview: Received message codestream/reporting/message(wv:39:GKlSLJB6RZyMUp8XH8KWdA) from the webview
[2019-05-17 19:08:04:716] AgentConnection.sendRequest(codestream/reporting/message) — {"extra":{"postId":"CJT219Z7X|1558117830.001800","streamId":"CJT219Z7X"},"message":"There was an error deleting a post: Request codestream/post/delete failed with message: Unable to find message with id=1558117830.001800","source":"webview","type":"error"}
[2019-05-17 19:08:04:719] WebviewPanel: Completed posting response(wv:39:GKlSLJB6RZyMUp8XH8KWdA) to the webview
[2019-05-17 19:08:05:550] Webview: Received message host/editor/range/highlight(wv:40:xMWIxgBUS4eUeZWVh4VAIA) from the webview
[2019-05-17 19:08:05:551] WebviewPanel: Completed posting response(wv:40:xMWIxgBUS4eUeZWVh4VAIA) to the webview
```

**Comment by planteater** [comment]
Thanks @mikestaub! Looks like the issue happens when you first delete the post from Slack, and then try to delete it from CodeStream. If you delete it from CodeStream it appears to work correctly, including triggering the deleting of the corresponding post on Slack. 

We'll get this fixed up asap.

**Comment by planteater** [comment]
Fixed in version 1.1.0
